bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>this block isn't straightforward. worth explaining at least two things:</p><p>1) what does the metadata filtering has to do with agent tools, how are those two combined to achieve some objective</p><p>2) what's the difference/relationship between this QueryEngineTool to the previews uber and lyft tools. (actually i didn't get it from reading it the first time)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463989'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>